.alexrc.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+exports.allow = [
+  // We frequently refer to form props by their name "disabled".
+  // Ideally we would alex-ignore only the valid uses (PRs accepted).
+  "invalid",
+
+  // Unfortunately "watchman" is a library name that we depend on.
+  "watchman-watchwoman"
+];
+
+// Use a "maybe" level of profanity instead of the default "unlikely".
+exports.profanitySureness = 1;
```

Comment:
When this is set to 0 it is too eager, it catches words like "color", "execute", and "fire" which in our context are super unlikely to be profane and highly likely to be valid

Reply:
Should you put your reasoning as a comment in the code?